app/cards/page.tsx

```diff
@@ -3,51 +3,69 @@ import Link from "next/link";
 import { useState, useEffect } from "react";
 import { Card } from "../../domains/cards/types";
 import { Button } from "@/components/uikit/button";
+import { deleteCard } from "@/domains/cards/services";
 
 export default function Cards(card: Card) {
   const [cards, setCards] = useState([]);
 
   useEffect(() => {
     let allCardsData = JSON.parse(localStorage.getItem("cards"));
     setCards(allCardsData);
-
-    console.log(allCardsData);
   }, []);
 
+  function handleDelete(id: string) {
+    deleteCard(id);
+    let allCardsData = JSON.parse(localStorage.getItem("cards"));
+    setCards(allCardsData);
+  }
+
   let sectionNode;
 
-  if (!cards) {
+  if (cards.length < 1) {
```

Comment:
```suggestion
  if (cards?.length < 1) {
```

`cards` може  бути `null`, тому тут помилка при відкритті сторінки.